ui/apps/platform/src/Components/PatternFly/IconText/IconText.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { ReactElement } from 'react';
+
+export type IconTextProps = {
+    Icon: ReactElement;
```

Comment:
nit: UPPERCASE prop name looks unusual

Reply:
Indeed, I will change to `icon`

Leftover from when I thought it would have element type and render as `<Icon />`